users/views.py

```diff
@@ -1,3 +1,41 @@
+import json, bcrypt
+
 from django.shortcuts import render
+from django.views     import View
+from django.http      import JsonResponse
+
+from users.models     import User
+from users.validation import validate_email, validate_password
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            user = json.loads(request.body)
```

Comment:
```suggestion
            data = json.loads(request.body)
```
request.body는 전달받은 data이지 user가 아닙니다. 오해의 소지가 있을 수 있으니 변수명 수정해주세요!